### PR TITLE
Library updates

### PR DESCRIPTION
**Pull request by visze**
- I removed unnecessary libraries in module poms.

- I update several libraries including guava, htsjdk (to latest 1.X version), slfj-api, antlr,...

- log4j is now used only in tests and in the clis modules.

- Include the maven enforcer update to find problems with different included versions of libraries.

### CODE REVIEW

File: jped-cli/pom.xml [hunk 1]
```diff
@@ -27,20 +27,20 @@
...
 		<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
-			<version>4.11</version>
-			<scope>test</scope>
-		</dependency>
-		<dependency>
 			<groupId>com.github.samtools</groupId>
 			<artifactId>htsjdk</artifactId>
-			<version>1.128</version>
```
Comment (holtgrewe) [inline jped-cli/pom.xml]: Indentation looks broken.

Could you verify that the indentation is consistent in all of your patches?

Comment (visze) [inline jped-cli/pom.xml]: seems to be ok

Comment (holtgrewe) [inline jped-cli/pom.xml]: Below it is broken

selection_005

Comment (visze) [inline jped-cli/pom.xml]: ok your right. but only the exclusion part

**Pull request merged**